sympy/core/operations.py

```diff
@@ -21,6 +25,20 @@ class AssocOp(Basic):
 
     This is an abstract base class, concrete derived classes must define
     the attribute `identity`.
+
+    Parameters
+    ==========
+
+    *args :
+        Arguments which are operated
+
+    sympify : bool, optional
```

Comment:
Perhaps this should be made private (`_sympify` and not documented). This is good for core library code, but I don't think it would be a good idea for most users to use this.